src/subsplit_dag.cpp

```diff
@@ -252,6 +252,18 @@ BitsetSizeMap SubsplitDAG::BuildEdgeIndexer() const {
   return edge_indexer;
 }
 
+SizeBoolVectorMap SubsplitDAG::BuildEdgeIdxToPCSPMap() const {
```

Comment:
By the name I'd expect this to take values in PCSP Bitsets, and here it's returning vector bool. I see that you're doing that so you can get it from Python, but perhaps changing the name a bit?

Reply:
Does `BuildEdgeIdxToPCSPBoolVectorMap` work?